### ISSUE TITLE
IPython Notebook Viewer after IPython.html has been deprecated

### ISSUE THREAD

**Issue by gph82** [mdtraj/mdtraj#927]
Hi, I'm trying to run this:
```python
from mdtraj.html import TrajectoryView, enable_notebook
enable_notebook()
```
from here http://mdtraj.org/latest/viewer.html, but I'm getting:
```python
/home/mi/gph82/anaconda/lib/python2.7/site-packages/IPython/html.py:14: ShimWarning: The `IPython.html` package has been deprecated. You should import from `notebook` instead. `IPython.html.widgets` has moved to `ipywidgets`.
  "`IPython.html.widgets` has moved to `ipywidgets`.", ShimWarning)

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-cc86fc55fde2> in <module>()
----> 1 from mdtraj.html import TrajectoryView, enable_notebook
      2 enable_notebook()

ImportError: cannot import name TrajectoryView
```

This is browser independent and reproduced in two Linux boxes. 
As for versions of stuff: 
conda list | grep -e jupyter -e ipython -e notebook -e mdtraj
ipython                   4.0.0                    py27_0  
ipython-genutils          0.1.0                     <pip>
ipython-notebook          4.0.4                    py27_0  
ipython-qtconsole         1.1.0                    py27_1  
ipython_genutils          0.1.0                    py27_0  
jupyter                   1.0.0                    py27_0  
jupyter-client            4.0.0                     <pip>
jupyter-console           4.0.2                     <pip>
jupyter-core              4.0.4                     <pip>
jupyter_client            4.0.0                    py27_0  
jupyter_console           4.0.2                    py27_0  
jupyter_core              4.0.4                    py27_0  
mdtraj                    1.4.2                np19py27_0  
notebook                  4.0.4                    py27_0

**Comment by rmcgibbo** [comment]
Is the warning definitely related to the exception? Does this error happen if you downgrade to IPython 3?

**Comment by gph82** [comment]
No error after downgrading, just the warning
":0: FutureWarning: IPython widgets are experimental and may change in the future."

**Comment by rmcgibbo** [comment]
Okay. So it looks like this is a IPython/Jupter v4 bug/incompatibility. The widgets API has been so unstable, that I'm sure we're out of sync somehow.

**Comment by 2grep** [comment]
Just adding, perhaps unsurprisingly, this is also an issue on OS X (same condo list output)

**Comment by mpharrigan** [comment]
was this fixed with #935?

**Comment by rmcgibbo** [comment]
Yes
